src/applications/simple-forms/mock-simple-forms-patterns/config/form.js

```diff
@@ -7,11 +7,11 @@ import ConfirmationPage from '../containers/ConfirmationPage';
 // pages
```

Comment:
For reviewers:
- The last 4 files are the relevant changes.
- The rest off the diff is cleanup/moving files/refactor mock form.